bigquery_etl/util.py

```diff
@@ -224,3 +292,60 @@ def coerce_result(*elements: Any) -> Generator[Any, None, None]:
             yield str(element)
         else:
             yield element
+
+
+def get_differences(exp, res, path="", sep=" / "):
```

Comment:
why was `" / "` chosen as the path separator here, instead of `"."`? this is for testing BigQuery SQL, so field names are guaranteed to not contain `.`, and `.` is an existing standard path separator, both for BigQuery and `jq`